forgae-shape-vue-command.md

```diff
@@ -0,0 +1,241 @@
+```
+
+## Deploy the contract
+
+Deploying the contract on the already started local network is done with only one command:
+
+```
+forgae deploy
+```
+
+The **deploy** command help developers run their deployment scripts for aeternity proejcts. The deploy script is scaffolded in deployment folder. We can see the ```ToDoManager.aes``` contract in contracts folder.
+```
+├── contracts
+    └── ToDoManager.aes
+├── deployment
+    └── deploy.js
+```
+
+The expected output of the command execution is:
+```
+===== Contract: ToDoManager.aes has been deployed =====
+Your deployment script finished successfully!
+```
+
+## Deployment details
+
+Let's get the deployment details with the following command:
+```
+forgae history
+```
+As a result, we can see a table with the most important details. For example:
+
+```
+┌───────────────┬──────────────────────────────────────────────────────┐
+│ Event Time    │ 2 May, 15:05:17                                      │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Executor      │ Deployer                                             │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Name or Label │ ToDoManager                                          │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Tx Hash       │ undefined                                            │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Status        │ Success                                              │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Gas Price     │ 1000000000                                           │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Gas Used      │ 4000                                                 │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Result        │ ct_HVb6d4kirgqzY1rShmzRTRwukcsXobjHcpLVD2EggoHmn6wt2 │
+├───────────────┼──────────────────────────────────────────────────────┤
+│ Network ID    │ ae_devnet                                            │
+└───────────────┴──────────────────────────────────────────────────────┘
+```
+Copy the address of the contract that is shown as a **Result** in the above report table.
+Assign the deployed contract address to the contractAddress property in src/contractDetails.js file.
+
+## Run the aepp
+Now we are ready to run the aepp.
+
+```
+cd aepp-forgae-shape-vue
+npm run serve
+```
+*By default the front-end app running at: Local: http://localhost:8080/*
+
+In order to test the aepp during development, please install this [Chrome plugin](https://chrome.google.com/webstore/detail/moesif-orign-cors-changer/digfbfaphojjndkpccljibejjbppifbc) (*or similar plugin for the alternative browsers*) which allows you to send cross-domain requests. Turn on the extension and test the aepp. For security reasons, please turn off after testing.
```

Comment:
the issue should be properly fixed in forgae. `localhost` is the same origin, the issue is just that allow `*` is set twice.